File: src/main/java/com/trade/securities/domain/StockData.java
package com.trade.securities.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class StockData {
    private String basDd; // 기준일자
    private String isuCd; // 종목코드
    private String isuNm; // 종목명
    private String mktNm; // 시장구분
    private String sectTpNm; // 소속부
    private String tddClsprc; // 종가
    private String cmpprevddPrc; // 대비
    private String flucRt; // 등락률
    private String tddOpnprc; // 시가
    private String tddHgprc; // 고가
    private String tddLwprc; // 저가
    private String accTrdvol; // 거래량
    private String accTrdval; // 거래대금
    private String mktcap; // 시가총액
    private String listShrs; // 상장주식수
}

File: src/main/kotlin/com/trade/securities/domain/StockData.kt
package com.trade.securities.domain

data class StockData(
    val basDd: String? = null, // 기준일자
    val isuCd: String? = null, // 종목코드
    val isuNm: String? = null, // 종목명
    val mktNm: String? = null, // 시장구분
    val sectTpNm: String? = null, // 소속부
    val tddClsprc: String? = null, // 종가
    val cmpprevddPrc: String? = null, // 대비
    val flucRt: String? = null, // 등락률
    val tddOpnprc: String? = null, // 시가
    val tddHgprc: String? = null, // 고가
    val tddLwprc: String? = null, // 저가
    val accTrdvol: String? = null, // 거래량
    val accTrdval: String? = null, // 거래대금
    val mktcap: String? = null, // 시가총액
    val listShrs: String? = null  // 상장주식수
)
